src/utils/googleAnalytics.ts

```diff
@@ -0,0 +1,25 @@
+import { event, initialize, pageview } from "react-ga";
+import { REACT_APP_GA_MEASUREMENT_ID } from "secrets";
+
+export const initGoogleAnalytics = () => {
+  if (REACT_APP_GA_MEASUREMENT_ID) {
+    initialize(REACT_APP_GA_MEASUREMENT_ID, {
+      debug: true,
```

Comment:
why do we need debug in the production?

Reply:
fixed in a6791f318084063a5d3390e616bb4e6f71e7a55d